scio-spanner/src/main/scala/com/spotify/scio/spanner/package.scala

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio
+
+import com.google.cloud.spanner._
+import com.spotify.scio.io.Tap
+import com.spotify.scio.values.SCollection
+import org.apache.beam.sdk.io.gcp.spanner.{MutationGroup, SpannerConfig, SpannerIO}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.Future
+
+/**
+ * Main package for Spanner APIs. Import all.
+ *
+ * {{{
+ * import com.spotify.scio.spanner._
+ * }}}
+ */
+package object spanner {
+
+  /** Create SpannerConfig object. Only projectId, instanceId, and databaseId are set. */
+  object SpannerConfigBasic {
+
+    def apply(projectId: String,
+              instanceId: String,
+              databaseId: String): SpannerConfig = {
+
+      SpannerConfig.create
+        .withProjectId(projectId)
+        .withInstanceId(instanceId)
+        .withDatabaseId(databaseId)
+    }
+  }
+
+  /** Enhanced version of [[ScioContext]] with Spanner methods. */
+  implicit class SpannerScioContext(val self: ScioContext) extends AnyVal {
+
+    /**
+     * Read from Spanner table. Return [[com.spotify.scio.values.SCollection SCollection]]
+     * of `Struct`s.
+     */
+    def spannerFromTable(projectId: String,
```

Comment:
I implemented this method to stay consistent with the how the `BigtableScioContext` class is implemented. It contains two methods; one for exposing `BigtableOptions` and the other for exposing the expanded values. If we want to diverge from this consistency, ok. Just want to double confirm. Let me know.

From the perspective of an API consumer, I actually prefer having the values expanded. I feel like it encapsulates the most common use case and doesn't require the consumer to have to deal with the `SpannerConfig` class. Considering that `projectId`, `instanceId`, and `databaseId` are most likely to be provided via command line parameters, I don't think it makes sense to force the consumer to go through one more step of initializing a `SpannerConfig` object. However, we should always provide the ability to do so if they require functionality outside of the common use case: `spannerFromTableWithConfig()`.

Thoughts?

Reply:
`Bigatable` is kinda the only one that does it iirc, and I just feel that, since config is so simple it's frictionless for users to implement and by "grouping" them, helps with readability and to some extent "separation of concerns" argument wise since we have a long method call instead of fluent api.

But, no strong feelings here 😄, this is not blocker for now!